test/src/completions.test.ts

```diff
@@ -16,15 +16,53 @@
 
 import * as assert from 'assert'
 import * as vscode from 'vscode'
-import { getTestDocUri, init, open, typeText, addFile } from './util'
+import { getTestDocUri, init, open, typeText, addFile, deleteFile, copyFile } from './util'
 
 suite('Completions', () => {
   const docUri = getTestDocUri('src/Temp.flix')
+  let tempDocUri: vscode.Uri | null = null
 
   suiteSetup(async () => {
     await init('completions')
   })
 
+  teardown(async () => {
```

Comment:
What's this? Is it necessary? I don't believe we do this for other tests.

Reply:
Hm. I modelled this after how it was done in the diagnostics suite. We specifically want there to only be one file in `src` at a time (even within the same suite), because the error produced by the unfinished line in one file can prevent any suggestions from appearing in the other. I at least know that some interference between tests is happening, and I assume this is it. For instance, when testing for completion suggests in `JavaMath`, if `JavaStringBuilder` also exists in `src`, the test fails. I suppose that is also an error worth addressing, since the suggestions should work either way, but it feels tangential to the issue I wanted to test.

Reply:
Oh, yes, sorry, I missed that you asked what it was. I saw this done in the diagnostics suite, as I said. From context, I assume this is run at the end of each test. This way, the `src` folder is always clean upon the start of a test.

Reply:
But I suppose I could move it to the inner test suite, since it's only relevant there

Reply:
@sockmaster27 proposed looking towards the diagnostics suite, so he can probably tell whether I've understood the trick used there and if I've applied it correctly here :)

Reply:
Maybe its OK. I am not super familiar with the codebase. Lets see what Holger says.

Reply:
I am always paranoid about deleteFile :)

Reply:
Damn, ofc. Tbh, it didn't occur to me before, but I can understand your paranoia. If it's not necessary, it's ofc. best to avoid :)

Reply:
This is perfectly in line with the way it's done in diagnostics. 
In `files.test.ts` I've done more or less the same thing by (admittedly inconsistently) running `init()` in `setup()` instead of `suiteSetup()` to hard reset the contents of the entire workspace before each test. I suppose this is the more declarative way to handle it, but it's not super important to me either way.

Reply:
Nice, good to hear. I wouldn't mind changing it to the files.test.ts approach though, if you think it's preferable :)